contributing_to_docs/term_glossary.adoc

```diff
@@ -214,6 +214,15 @@ instance / cluster as an "OpenShift deployment".
 
 Do not use: deployment configuration(s), deployment config(s)
 
+''''
+=== disconnected
+
+Usage: disconnected environment, disconnected installation
+
+Use "disconnected" when discussing installing a cluster in an environment that does not have an active connection to the internet. Use "disconnected" regardless of whether the restriction is physical or logical.
```

Comment:
I'd like additional sentences aimed at readers and authors similar to:

> In many cases, particular cluster behavior depends whether particular URIs are accessible from within the cluster.  It is possible for some URIs to be accessible, with their consuming components functioning in an internet-connected manner, while other URIs are not accessible, with their consuming components functioning in a disconnected manner.

And then something aimed at authors similar to:

> To reduce ambiguity, documentation discussion disconnected behavior should explicitly list relevant URIs in their prerequisites section.

Or maybe those prerequisites belong in the connected-behavior sections, and the disconnected-behavior sections are where readers fall through to when the connected-behavior prereqs are not satisfied?

Reply:
@wking Sorry just coming back to this now. I honestly doubt that it would be feasible for docs to list which URIs are required in each section related to disconnected docs, or even in the connected ones. I don't know how we'd find this information, and I'm guessing it could easily change and we wouldn't know about it to update the docs.

Unless you have a strong objection to using "disconnected" as the preferred term, let's move forward with this as our stance. And if there are things that we can do otherwise to improve the experience for users on knowing whether to follow the connected or disconnected sections, let's follow up on that separately, if that's something we want to pursue. Does that sound okay?